man/sd_listen_fds.xml

```diff
@@ -198,6 +198,9 @@
     number. <function>sd_listen_fds_with_names()</function> does the
     same but also parses <varname>$LISTEN_FDNAMES</varname> if
     set.</para>
+
+  <para>Services that specify <varname>StandardInput=socket</varname> are not meant to use
+  these functions as their environment is free of <varname>$LISTEN_FDS</varname> variable.</para>
```

Comment:
"as the ... variable is not set in their environment" sounds better

Reply:
fixed.